digit-ui-esbuild/products/dashboard/src/styles/dashboard.css

```
@@ -4132,6 +4145,21 @@
 
+/* Screen-reader-only header row inside the scrollable body table — restores
+   column semantics lost by the pinned-header split (DashboardTableFrame). */
+
+.dashboard-table-sr-head {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)